### PR TITLE
[Portuguese] Translate guide/api.md

### PR DESCRIPTION
**Pull request by roziscoding**

### CODE REVIEW

File: site/docs/pt/guide/api.md [hunk 1]
```diff
@@ -0,0 +1,136 @@
+# Bot API
+
+## Informações Gerais
+
+Os bots do Telegram se comunicam com os servidores do Telegram por meio de solicitações HTTP.
+A Bot API do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```
Comment (kr4t0z) [inline site/docs/pt/guide/api.md:6]:
```suggestion
A API de bots do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```

File: site/docs/pt/guide/api.md [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+Ela define tudo o que os bots do Telegram podem fazer.
+Você pode encontrá-la vinculada na aba Recursos.
+
+A configuração pode ser visualizada da seguinte forma:
+
+```asciiart:no-line-numbers
+( ( ( Telegram ) API MTProto ) API HTTP de Bots ) <-- o bot se conecta aqui
+```
+
+Em palavras: quando seu bot envia uma mensagem, ela será enviada como uma solicitação HTTP para um _servidor da Bot API_ (seja hospedado pela equipe do Telegram ou [hospedado por você](https://core.telegram.org/bots/api#using-a-local-bot-api-server)).
```
Comment (kr4t0z) [inline site/docs/pt/guide/api.md:16]:
```suggestion
Em palavras: quando seu bot envia uma mensagem, ela será enviada como uma solicitação HTTP para um _servidor da API de bots_ (seja hospedado pela equipe do Telegram ou [hospedado por você](https://core.telegram.org/bots/api#using-a-local-bot-api-server)).
```

File: site/docs/pt/guide/api.md [hunk 3]
```diff
@@ -0,0 +1,136 @@
+# Bot API
+
+## Informações Gerais
+
+Os bots do Telegram se comunicam com os servidores do Telegram por meio de solicitações HTTP.
+A API de bots do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```
Comment (kr4t0z) [inline site/docs/pt/guide/api.md:6]:
```suggestion
A Bot API do Telegram é a especificação dessa interface, ou seja, uma [longa lista](https://core.telegram.org/bots/api) de métodos e tipos de dados, comumente chamada de referência.
```

File: site/docs/pt/guide/api.md [hunk 4]
```diff
@@ -0,0 +1,136 @@
...
+Em palavras: quando seu bot envia uma mensagem, ela será enviada como uma solicitação HTTP para um _servidor da Bot API_ (seja hospedado pela equipe do Telegram ou [hospedado por você](https://core.telegram.org/bots/api#using-a-local-bot-api-server)).
+Este servidor irá traduzir a solicitação para o protocolo nativo do Telegram chamado MTProto e enviar uma solicitação para o backend do Telegram, que cuida do envio da mensagem para o usuário.
+
+Analogamente, quando um usuário responde, o caminho inverso é percorrido.
+
+::: tip Contornando os Limites de Tamanho de Arquivo
+O backend do Telegram permite que seu bot [envie arquivos](./files) de até 2000 MB.
+No entanto, o servidor da Bot API responsável por traduzir as solicitações para HTTP restringe o tamanho do arquivo para 50 MB para downloads e 20 MB para uploads.
+
+Portanto, se você contornar o servidor da Bot API que o Telegram executa para você e simplesmente [hospedar seu próprio servidor da Bot API](https://core.telegram.org/bots/api#using-a-local-bot-api-server), você pode permitir que seu bot envie arquivos de até 2000 MB.
```
Comment (kr4t0z) [inline site/docs/pt/guide/api.md:25]:
```suggestion
Portanto, se você contornar o servidor da Bot API que o Telegram fornece para você e simplesmente [hospedar seu próprio servidor da Bot API](https://core.telegram.org/bots/api#using-a-local-bot-api-server), você pode permitir que seu bot envie arquivos de até 2000 MB.
```

File: site/docs/pt/guide/api.md [hunk 5]
```diff
@@ -0,0 +1,136 @@
...
+}
+```
+
+Em alguns casos, isso também pode funcionar sem ajustar sua configuração do TypeScript.
+
+::: aviso Autocomplete Incorreto
+
+Se você não alterar seu arquivo `tsconfig.json` conforme descrito acima, pode acontecer de seu editor de código sugerir, no autocomplete, importar os tipos de `grammy/out/client` ou algo do tipo.
+**Todos os caminhos começando com `grammy/out` são internos. Não os utilize.**
+Eles podem ser alterados arbitrariamente a qualquer momento, então recomendamos fortemente que você importe de `grammy/types` em vez disso.
```
Comment (kr4t0z) [inline site/docs/pt/guide/api.md:115]:
```suggestion
Eles podem ser alterados arbitrariamente a qualquer momento, então recomendamos fortemente que você importe de `grammy/types`.
```

**Review by kr4t0z — CHANGES_REQUESTED** [review]
Just minor issues, everything else are good.

File: site/docs/pt/guide/api.md (hunk 3 above)
Comment (roziscoding) [reply site/docs/pt/guide/api.md:6]: In this case I think we can keep it as-is, because we're specifying that it's Telegram's Bot API

**Review by kr4t0z — APPROVED** [review]
LGTM

**Pull request merged**